devel/libecl/include/ert/ecl/ecl_kw_magic.h

```diff
@@ -290,6 +290,7 @@ extern "C" {
 #define NNCHEADA_ILOC1_INDEX 0     /*ILOC1: Index of first LGR*/
 #define NNCHEADA_ILOC2_INDEX 1     /*ILOC2: Index of second LGR*/
 #define NNA_NUMNNC_INDEX     0     /*Item 1 in NNA1 or NNA2 is number of NNCs*/
+#define NNCHEAD_SIZE        10
```

Comment:
Add `/* NUMNNC LGR (0 for global) followed by eight 0's (placeholders)*/` or similar after `10`?